lib/que/poller.rb

```diff
@@ -63,6 +63,7 @@ class Poller
           SELECT
             (j).*,
             l.locked,
+            extract(epoch from (now() - (j).run_at)) as latency,
```

Comment:
> NB, using `now() - (j).run_at` as latency/delay can be misleading. If a job errors and retries with a backoff then the `run_at` changes. It would be best to add an "initial_run_at" column to the main jobs table, but that would be a better change....

@hlascelles This is a good point. I think the intention here is to permit ascertaining worker contention - by allowing you to analyse at the delay between when a job is desired to be run at (or re-run at), and when it's actually picked up to be run. In terms of that desire, I think this metric is more useful. But I'm sure there's also merit in https://github.com/que-rb/que/issues/325.